README.md

```diff
@@ -32,10 +32,14 @@ in a single K8s cluster.
 * A user with administrative privileges in the OpenShift cluster and is logged in using `oc` or `kubectl`
 * brew coreutils (Mac only)
 * OSD Cluster with the following specs:
-   * 3 compute nodes
-   * Size: m5.4xlarge
-   * MultiAz: True
-
+   * Plan `developer.x1` (clusters with less resources _may_ work, but have not been verified with kas-installer)
+      * 3 compute nodes
+      * Size: r5.2xlarge
+      * MultiAz: N/A
+   * Plan `standard.x1`
+      * 3 compute nodes
+      * Size: m5.4xlarge
```

Comment:
Currently, I'm using 9 nodes of type `m5.2xlarge` for 3 broker Kafka Instance (`standard.x1`) and 12 nodes of type `m5.2xlarge` for 6 brokers Kafka Instance (`standard.x2`). The reason of using `m5.2xlarge` instead of `m5.4xlarge` is to use the same CPU type as prod and staging environment. What do you think about recommending this other configuration?

Reply:
@agullon , good suggestion, I'll update the PR. The m5.4xlarge is from the original docs, so updating to the current approach is a good idea.

Reply:
What I have noticed is having different suggestions from different usage just leads to different expectations, as in this case how many `m5.4xlarge` we need for 3 brokers and how many for 6 brokers or dev instance etc.  Keeping them aligned I would recommend. On the flip side, the calculations for AWS spending are based on what is used stage/prod. a skew might not work in our favor to curb the costs, unless we see that it is actually saving money

Reply:
@rareddy are you thinking to look into the actual minimums to get things working for each profile or leave this section completely out?

Reply:
we did the minimum sizing calculations as part of InstanceProfiler work before that is where we came up with 
* 9 `m5.2xlarge` for `standard.x1` 
* 12 `m5.2xlarge` for `standard.x2`
* 5 or 6 nodes I think for `developer.x1`, but here you could place more than one instance.

we do not use `m5.4xlarge` so IMO it is easier to keep it out of it.

Reply:
Updated with 6, 9, and 12 `m5.2xlarge` nodes.